### ISSUE TITLE
Binance - There was an error requesting exchange info 

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/hummingbot#6062]
**Describe the bug**
We have observed an error related to Binance `API rate limit on REQUEST_WEIGHT (1200 calls per 60s) has almost reached`. As per initial investigation, the issue seems from the `rate_oracle_source which` is set to default `binance`. Currently we are investigating this issue and update shall be added on this ticket.

On the screenshot below, the bot was using Kucoin yet the issue for Binance API rate limit still occurs
![Screenshot 2023-02-06 at 8 11 37 PM](https://user-images.githubusercontent.com/73840223/216969032-bd75a8bb-87c4-4c33-9874-60c909537597.png)

```
2023-02-06 12:00:47,643 - 31693 - hummingbot.core.api_throttler.async_request_context_base - INFO - API rate limit on REQUEST_WEIGHT (1200 calls per 60s) has almost reached. Limits used is 1193 in the last 60 seconds
2023-02-06 12:01:17,690 - 31693 - hummingbot.core.api_throttler.async_request_context_base - INFO - API rate limit on REQUEST_WEIGHT (1200 calls per 60s) has almost reached. Limits used is 1193 in the last 60 seconds
2023-02-06 12:02:49,310 - 31693 - hummingbot.connector.exchange.binance.binance_exchange.BinanceExchange - ERROR - There was an error requesting exchange info.
Traceback (most recent call last):
  File "/home/ubuntu/kucoin/hummingbot/connector/exchange_py_base.py", line 1037, in _initialize_trading_pair_symbol_map
    exchange_info = await self._api_get(path_url=self.trading_pairs_request_path)
  File "/home/ubuntu/kucoin/hummingbot/connector/exchange_py_base.py", line 854, in _api_get
    return await self._api_request(*args, **kwargs)
  File "/home/ubuntu/kucoin/hummingbot/connector/exchange_py_base.py", line 896, in _api_request
    request_result = await rest_assistant.execute_request(
  File "/home/ubuntu/kucoin/hummingbot/core/web_assistant/rest_assistant.py", line 75, in execute_request
    raise IOError(f"Error executing request {method.name} {url}. HTTP status is {response.status}. "
OSError: Error executing request GET https://api.binance.us/api/v3/exchangeInfo. HTTP status is 429. Error: {"code":-1003,"msg":"Too much request weight used; current limit is 1200 request weight per 1 MINUTE. Please use the websocket for live updates to avoid polling the API."}

```

For the workaround, please change the `rate_oracle_source` to any of the available list `ascendex, kucoin, gateio and coingecko`. In the event that of 0 balances is displayed, change the `global_token.global_token_name` to `USDT` 

**Steps To Reproduce**
1. Setup any Hummingbot strategy
2. Connect to any available connector 
3. Start the client and observed that it should be displaying the issue on output and log panel 

**Release version**
v1.12.0 and development branch

**Attachments**
[logs_pmm-ooeusdt.log](https://github.com/hummingbot/hummingbot/files/10662909/logs_pmm-ooeusdt.log)

**Comment by roman-hrybinchuk** [comment]
+1

**Comment by nikspz** [comment]
Fixed in 1.15.0 Release
